openff/toolkit/typing/engines/smirnoff/parametertypes.py

```diff
@@ -0,0 +1,46 @@
+"""Re-exports of concrete ParameterTypes
+
+openff.toolkit.typing.engines.smirnoff.parameters defines a number of parameter
+types within class definitions of the corresponding ParameterHandler. This module
+re-exports them for discoverability and ease of use."""
```

Comment:
(blocking) Per our discussion today, let's 

* delete this file
* move the exporting of the `ParameterTypes` to `parameters.py` itself
* include in exported `ParameterTypes` in the `__all__` for `parameters.py`
* Add a unit test to `test_parameters` that ensures that all ParameterType subclasses owned by ParameterHandler subclasses are exposed via re-export and are in `__all__`